treeherder/config/settings.py

```diff
@@ -435,6 +435,10 @@
 BUGFILER_API_URL = env("BUGZILLA_API_URL", default=BZ_API_URL)
 BUGFILER_API_KEY = env("BUGZILLA_API_KEY", default=None)
 
+# Auth0 setup
+AUTH0_DOMAIN = env('AUTH0_DOMAIN', default="auth.mozilla.auth0.com")
+AUTH0_AUDIENCE = env('AUTH0_AUDIENCE', default="login.taskcluster.net")
```

Comment:
Does the audience of `login.taskcluster.net` mean that cross-origin Django logins (either `localhost:5000` UI -> `localhost:8000/api`, or `localhost:5000` UI -> `treeherder.mozilla.org/api`) will work, since it will be the same for all environments?

If so, that will be awesome :-)

Reply:
This part will, yes.  The `access_token` will contain that audience, indicating to the login service that credentials should be granted corresponding to the `access_token`.  Regardless of origin.

Reply:
Now that I look again, this value should be used on the frontend when initiating an Auth0 authentication, but should not appear anywhere on the backend.